tutorials/users/ide/kernels-and-dependency-management.md

```diff
@@ -1 +1,85 @@
-# kernels-and-dependency-management
+# Kernels and Dependency Management
+
+## What is a Kernel?
+
+Kernels in Jupyter are self-contained, program language specific processes that run independently.  
+Having kernels configured and packaged for specific purposes gives the Jupyter environment some flexibility.
+A kernel doesn't need to be configured to serve every purpose imaginable. Several kernels can be made, with none
+of the dependencies for one kernel affecting the dependencies for another. If a kernel needs to be updated, changes can 
+be made without affecting other kernels.  If it is no longer needed, it can be deleted without 
+impacting other kernels.
+
+Below are examples of kernels available on Marble.  These will be updated or changed to fit the needs of the users.
+
+![Kernels](images/version-control/jupyter-notebook-buttons.png)
```

Comment:
This image contains the "Notebook" header at the top. This may be confusing since this image is intended to show how to create a new notebook. I would use an image that just shows the two kernel buttons without the "Notebook" header.

Reply:
Changed image to one without "Notebook" header